train_rl4lm_allrates.py

```diff
@@ -22,38 +28,39 @@
     rate: {rate}
```

Comment:
to change the toy task in the datapool, you can pass the `toy` argument, (integer)